lib/matplotlib/collections.py

```diff
@@ -545,7 +545,10 @@ def set_offsets(self, offsets):
         ----------
         offsets : (N, 2) or (2,) array-like
         """
-        offsets = np.asanyarray(offsets, float)
+        offsets = np.asanyarray(offsets)
+        offsets[:, 0] = self.convert_xunits(offsets[:, 0])
+        offsets[:, 1] = self.convert_xunits(offsets[:, 1])
```

Comment:
?
```suggestion
        offsets[:, 1] = self.convert_yunits(offsets[:, 1])
```

Reply:
Ooops, yes, fixed.  The conversion also had to take place after co-ercing to 2xN...